src/3b_count_aca.jl

```diff
@@ -43,15 +51,21 @@ end
 # Loop over each reef and count number of slopes and flats that meet criteria
 @showprogress dt = 10 desc = "Collating zonal stats..." for reg in REGIONS
     # Load rasters and identify cells that are greater than or equal to 0.95
-    target_flats = Raster(joinpath(ACA_OUTPUT_DIR, "$(reg)_suitable_flats.tif"))
+    target_flats = Raster(
+        joinpath(ACA_OUTPUT_DIR, "$(reg)_suitable_flats.tif"),
+        crs=EPSG(7844),
+        lazy=true
+    )
     target_flats = read(target_flats .>= 0.95)
 
-    target_slopes = Raster(joinpath(ACA_OUTPUT_DIR, "$(reg)_suitable_slopes.tif"))
+    target_slopes = Raster(
+        joinpath(ACA_OUTPUT_DIR, "$(reg)_suitable_slopes.tif"),
+        crs=EPSG(7844),
+        lazy=true
+    )
     target_slopes = read(target_slopes .>= 0.95)
 
-    # Reproject reef features to maintain consistent crs with raster files
-    reefs = GDF.read(REEF_PATH)
-    reefs.geometry = AG.reproject(reefs.geometry, GFT.EPSG(4326), crs(target_flats); order=:trad)
+    reefs = reef_features
```

Comment:
I assume this is necessary to iterate over for loop below. Otherwise not neessary

Reply:
Yeah it's necessary to get valid output values but I'm not sure why that's the case.